.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,56 @@
+name: unit tests
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/unit_tests.yml
+      - '**.py'
+      - '**.html'
+      - '**.jinja'
+      - '**.js'
+      - poetry.lock
+      - pyproject.toml
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/unit_tests.yml
+      - '**.py'
+      - '**.html'
+      - '**.jinja'
+      - '**.js'
+      - poetry.lock
+      - pyproject.toml
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.7
+        with:
+          submodules: 'true'
```

Comment:
Just curious about this line. Which submodules are being checked out?

Reply:
The https://github.com/VectorInstitute/cyclops-webpage repo is used for the landing page. This is built using docusaurus. I guess we don't need this for the unit test workflow. Just the docs ones. Will update!

Reply:
Gotcha. Thanks!